src/Messages.sol

```diff
@@ -38,7 +38,9 @@ library ConnectorMessages {
         /// 13 - Collect investment
         CollectInvest,
         /// 14 - Collect Redeem
-        CollectRedeem
+        CollectRedeem,
+        /// 15 - Schedule a spell to be relied on gateway (Admin only)
+        ScheduleAddAdmin
```

Comment:
Thinking about this more, I would change this to
```suggestion
        AddAdmin
```
since scheduling is an implementation detail. Theoretically, if we deployed Connectors on another parachain with XCM as a fully trustless bridge, this wouldn't need any delay at all.